src/shiv/bootstrap/__init__.py

```diff
@@ -14,6 +16,20 @@
 from .interpreter import execute_interpreter
 
 
+def run(module):
+    """Run a module in a scrubbed environment."""
+
+    # delete the entry point/script env vars
+    #
+    # If a single pyz has multiple callers, we want to remove these vars as we no longer need them
+    # and they can cause subprocesses to fail with a ModuleNotFoundError.
+    with suppress(KeyError):
+        del os.environ[Environment.MODULE]
+        del os.environ[Environment.ENTRY_POINT]
```

Comment:
Should this be split across two "with suppress" calls?

Reply:
Either `del` call could raise KeyError and we don't particularly care if they do, so I don't think so. No need to make two function calls when one is sufficient.